client/my-sites/comments/comment-list/index.jsx

```diff
@@ -93,7 +103,11 @@ export class CommentList extends Component {
 
 	isCommentSelected = commentId => !! this.state.selectedComments[ commentId ];
 
-	isSelectedAll = () => this.props.comments.length === size( this.state.selectedComments );
+	isSelectedAll = () => this.getComments().length === size( this.state.selectedComments );
```

Comment:
is `selectedComments` not an array? it's a little odd seeing two different ways of computing collection size right next to each other.